zerver/views/user_groups.py

```diff
@@ -115,6 +119,54 @@ def update_user_group_backend(
     return json_success(request, data)
 
 
+def send_messages_for_user_groups(
+    acting_user: UserProfile,
+    recipient_users: List[UserProfile],
+    user_group_name: str,
+    *,
+    send_subscription_message: bool,
+    send_unsubscription_message: bool,
```

Comment:
I like the use of mandatory kwargs, but it might be better for both of these to default to false, so that callers only need to pass `send_subscription_message=True` if they want that behavior.